db/schema.rb

```diff
@@ -49,7 +49,18 @@
     t.citext   "name"
   end
 
+  create_table "transactions", force: :cascade do |t|
+    t.integer  "invoice_id"
+    t.integer  "credit_card_number"
```

Comment:
Since we're not doing any math on these credit card numbers (multiply/divide/compare to...), we could store it as a string instead.